components/ws-daemon/pkg/daemon/config.go

```diff
@@ -30,9 +29,3 @@ type RuntimeConfig struct {
 	Kubeconfig          string            `json:"kubeconfig"`
 	KubernetesNamespace string            `json:"namespace"`
 }
-
-type ReadinessSignalConfig struct {
-	Enabled bool   `json:"enabled"`
-	Addr    string `json:"addr"`
-	Path    string `json:"path"`
-}
```

Comment:
Cool, we can remove from post processing later (for deploys from ops)?

Reply:
already done